Examples/Algorithms/Geant4/include/ActsExamples/Geant4/RegionCreator.hpp

```diff
@@ -0,0 +1,68 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Utilities/Logger.hpp"
+
+#include <string>
+#include <vector>
+
+namespace ActsExamples {
+
+/// Geant4 Region Creator
+class RegionCreator {
+ public:
+  /// Nested configuration struct for the Geant4 region creator
+  struct Config {
+    /// Cut to be applied for gammas
+    double gammaCut{};
+
+    /// Cut to be applied for electrons
+    double electronCut{};
+
+    /// Cut to be applied for positrons
+    double positronCut{};
+
+    /// Cut to be applied for protons
+    double protonCut{};
+
+    /// Volume list to be included in this region
+    std::vector<std::string> volumes{};
+  };
+
+  /// Region creator constructor
+  ///
+  /// @param cfg is the configuration struct
+  /// @param name is the region name
+  /// @param level is the logging level to be used
+  RegionCreator(const Config& cfg, std::string name,
+                Acts::Logging::Level level);
+  ~RegionCreator() {}
```

Comment:
```suggestion
```

I think the destructor definition can be dropped or use `= default`

Reply:
Done.